### ISSUE TITLE
Algolia search not returning matches

### ISSUE THREAD

**Issue by kubu4** [mmistakes/jekyll-theme-basically-basic#78]
I've configured Algolia search with GitHub Pages via Travis CI.

I can see in the Algolia dashboard that my site is being indexed:

![20181029_001](https://user-images.githubusercontent.com/4514104/47668014-9d798700-db64-11e8-9969-64c20a3f58ea.png)

---

Searching my site (with a matching string) doesn't return any results, nor does it say "No results":

![20181029_002](https://user-images.githubusercontent.com/4514104/47668086-c8fc7180-db64-11e8-80ba-790821e032a4.png)

---

However, if I search with a string that _doesn't_ match any string in my site, I do get the "No results" message:

![20181029_003](https://user-images.githubusercontent.com/4514104/47668237-2264a080-db65-11e8-98bc-aa5b6e274e43.png)

---

Finally, if I type a matching string in the search box _after_ I've entered in a non-matching string, the "No results" message persists and I still get no search results:

![20181029_004](https://user-images.githubusercontent.com/4514104/47668783-99e6ff80-db66-11e8-85d8-25840aadd5b0.png)

---

## Environment

- Basically Basic version: 1.4.1
- Ruby gem or remote theme version: 
- Jekyll version:
- Git repository URL: https://github.com/RobertsLab/sams-notebook
- GitHub Pages hosted (if yes provide URL to site): https://robertslab.github.io/sams-notebook/
- Operating system: Ubuntu 16.04
- Browser: Firefox

## Expected behavior

Return matching search results.

**Comment by mmistakes** [comment]
I've seen this before on my personal site. Don't think it has anything to do with the theme.

If you log into your Algolia account and click on the index, there are many configurations you can make. I've never messed with them, but there are all sorts of things related to what sort of attributes are indexed, returned, synonyms, stop words, typos, etc etc etc.

It probably needs to be configured some.

**Comment by kubu4** [comment]
OK, thanks for the info.

Also, I wouldn't be surprised if it was related to Issue #81. Haven't had time to test since I made those changes.
